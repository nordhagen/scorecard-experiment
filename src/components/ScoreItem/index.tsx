import * as React from 'react'
import styles from './style.scss'

interface Props {
  id: string
  qty: number
  score: number
}

const ScoreItem : React.FunctionComponent<Props> = props => {
  return (
    <tr className={styles.ScoreItem}>
      <td>{props.id}</td>
      <td>{props.qty}</td>
      <td>{props.score}</td>
    </tr>
  )
}

export default ScoreItem
