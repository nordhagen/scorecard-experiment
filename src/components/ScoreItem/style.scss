@import '../../global.scss';

.EPGChannelItem {
  display: flex;
  flex-direction: row;
  height: 100px;

  @include media(phablet) {
    border-top: 1px solid #333;
  }

  h2 {
    display: block;
    position: absolute;
    width: 100px;
    height: 100px;
    background-color: black;
    background-position: 50%;
    background-repeat: no-repeat;
    background-size: 70%;
    text-indent: -1000px;
    margin: 0;
  }

  .scheduleList {
    display: block;
    width: 100%;
    margin: 0 20px 0 100px;
    position: relative;

    @include media(phablet) {
      width: 100%;
      white-space: nowrap;
      overflow: auto;
      margin-right: 0;
      max-width: 100000px;
    }

    &:before {
      content: 'Nothing on right now';
      position: absolute;
      display: flex;
      height: 100%;
      width: 100%;
      flex-direction: column;
      justify-content: center;
      z-index: 1;
      color: #333;
    }
  }
}
