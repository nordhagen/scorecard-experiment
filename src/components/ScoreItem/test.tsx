import React from 'react'
import ScoreItem from './index'
import renderer from 'react-test-renderer'

test('It renders correctly', () => {
  const pastItem = renderer.create(<ScoreItem id="a" qty={1} score={50} />)

  expect(pastItem.toJSON()).toMatchSnapshot()
})
