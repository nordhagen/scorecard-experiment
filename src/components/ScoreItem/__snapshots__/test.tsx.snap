// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
<tr>
  <td>
    a
  </td>
  <td>
    1
  </td>
  <td>
    50
  </td>
</tr>
`;
